src/plugins/Quote.js

```diff
@@ -5,6 +5,14 @@ const Plugin = require("../Plugin");
 
 module.exports = class Quote extends Plugin {
 
+    constructor(...args) {
```

Comment:
This is a good idea, using the constructor makes more sense than `start` (although, maybe `stop` should then be renamed to something else).